docs/testing.md

```diff
@@ -0,0 +1,41 @@
+# Testing
+
+Based on upstreams documentation https://github.com/thtanaka/kubernetes/blob/master/docs/devel/testing.md we use three levels of testing: unit, integration and e2e.
+
+Running `make test` executes all the test suites.
+
+We use ginkgo for testing. Every package needs a `suite_test.go` for setup. It can be generated by running `ginkgo bootstrap` in the sub folder. Rename the generated file afterwards.
```

Comment:
Maybe explain why the file needs to be renamed? (it's not clear to me right now)

Reply:
We recently renamed all the existing files. Also all controllers are now just named `controller.go`. So it is mostly about consistency.